sqlflow/client.py

```diff
@@ -1,13 +1,50 @@
+import sys
 import os
 import logging
-
 import grpc
+
 import google.protobuf.wrappers_pb2 as wrapper
+from google.protobuf.timestamp_pb2 import Timestamp
 
 import sqlflow.proto.sqlflow_pb2 as pb
 import sqlflow.proto.sqlflow_pb2_grpc as pb_grpc
 
+
 _LOGGER = logging.getLogger(__name__)
+_OUT_HANDLER = logging.StreamHandler(sys.stdout)
```

Comment:
Still have concern to set log handler here. may we move it into magic command?

Reply:
Done.